pennylane/qchem/molecule.py

```diff
@@ -55,6 +55,10 @@ class Molecule:
         r (array[float]): positions of the Gaussian functions
```

Comment:
Hmm I didn't touch this in the PR. Should we remove this? What does it actually do?

Reply:
I know you didn't. We simply set it inside the `init` itself, but I guess it is documented here since it's used in other lower functions. So I think we can let it be for now.

Reply:
Okie dokie. Yeah I was just saying that IDK what this guy does so I don't want to accidentally remove something randomly lol